src/services/kubernetes/broker-deployment.yaml

```diff
@@ -19,3 +19,8 @@ spec:
             - containerPort: 5672
             - containerPort: 15672
             - containerPort: 15674
+          resources:
+            requests:
+              memory: "80M"
+              cpu: "100m"
```

Comment:
Should we give the broker more resources? It is the main bottleneck.

Reply:
You are right, the broker memory parameter is too low, let me fix that. The CPU value is a little bit higher, I think I swapped the values while editing 😅 . 
I could leave the CPU to 100, in that way we are always sure we will have the resources.